superset-frontend/src/pages/DatabaseList/index.tsx

```diff
@@ -230,6 +233,16 @@ function DatabaseList({
     {
       label: t('Upload file to database'),
       childs: [
+        {
+          label: t('New Upload CSV'),
+          name: 'Upload CSV file',
+          url: '#',
+          onClick: () => {
+            setCsvUploadModalOpen(true);
+          },
```

Comment:
is there a simple way to make this a button instead of an href?

Reply:
could not find an easy way without refactoring the component